src/madpack/madpack.py

```diff
@@ -39,6 +39,7 @@
 # (b) __file__ (instead of sys.argv[0]) because madpack.py could be called
 # (a) through a symbolic link and (b) not as the main module.
 maddir = os.path.abspath(os.path.dirname(os.path.realpath(__file__)) + "/..")   # MADlib root dir
+maddir_abs = os.path.abspath(os.path.dirname(os.path.abspath(__file__)) + "/../..")   # MADlib abs root dir
```

Comment:
The names `maddir` and `maddir_abs` are a bit confusing since they can be used interchangeably. Do you think there's a better way to name these variables ?